app/components/searchBar/index.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { screen, render, fireEvent, getByTestId, waitFor } from '@testing-library/react';
+import SearchBar from '.';
+import userEvent from '@testing-library/user-event';
+
+describe('Input Value', () => {
+  it('renders a input field', () => {
+    render(<SearchBar placeholder="Search" defaultValue="" onChangeValue={() => {}} />);
+    const searchBar = screen.getByTestId('searchBar');
+    expect(searchBar).toBeInTheDocument();
+  });
+});
+
+describe('Not Empty', () => {
+  it('checks input value to be empty', () => {
+    render(<SearchBar placeholder="" defaultValue="" onChangeValue={() => {}} />);
+    const inputElement = screen.getByTestId('searchBar');
+    expect(inputElement).toBeInTheDocument();
+  });
+});
```

Comment:
Have better descriptions under `describe` and `it` blocks

Reply:
Done